src/refiners/foundationals/latent_diffusion/model.py

```diff
@@ -89,6 +88,8 @@ def forward(
         self.set_unet_context(timestep=timestep, clip_text_embedding=clip_text_embedding, **kwargs)
 
         latents = torch.cat(tensors=(x, x))  # for classifier-free guidance
+        # scale latents for schedulers that need it
+        latents = self.scheduler.scale_model_input(latents, step=step)
```

Comment:
This won't fly if e.g. you don't modify the `Scheduler` base class with a NOP version. For example, with a vanilla SD (not using Euler scheduler):

```
AttributeError: 'DPMSolver' object has no attribute 'scale_model_input'
```

Reply:
Yes, the abtract class `scheduler` needs to have a naive `scale_model_input` that does nothing for the other schedulers. I will add it.